src/electionguard-ui/ElectionGuard.UI/Views/EncryptionExportPage.xaml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--  TODO: Look at making a popup instead of base page  -->
```

Comment:
This was stubbed as a copy of ballotUpload, since from a UI perspective, there's shared functionality